stylesheets/styles.css

```diff
@@ -308,6 +308,14 @@ a.github {
 	background: url(../images/octocat-small.png) no-repeat 1px;
 }
 
+a.clean {
+	-webkit-font-smoothing: antialiased;
+	font-weight: normal;
+	text-shadow: rgba(0, 0, 0, 0.4) 0 -1px 0;
+	padding: 2px 2px 2px 22px;
+  height: 30px;
```

Comment:
... or maybe it's some kind of button after all?!? Can I see a demo rendering of this somewhere? Why is this CSS named "clean"? It seems to add a shadow, which doesn't sound like "cleaning" to me?!?